### PR TITLE
Publish form

### PR DESCRIPTION
**Pull request by stephanebisson**
Test it here : https://wikimedia.github.io/wikistories-prototype/publish/#/story

### CODE REVIEW

File: src/views/Publish.vue [hunk 1]
```diff
@@ -17,10 +26,18 @@
...
+    import Navigator from '@components/Navigator.vue'
 
     export default {
         name: 'Publish',
-        components: { PrimaryButton },
+        data: () => {
+            const today = new Date();
+            const date = `${today.getFullYear()}-${today.getMonth()+1}-${today.getDate()}`
+            return {
+                date
```
Comment (stephanebisson) [inline src/views/Publish.vue:37]: Data is meant for private instance data that the component can read/write. This date field is more of a `computed`

File: src/views/Publish.vue [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    <div class="view publish">
+        <Navigator :onBack="() => this.$router.push( { name: 'Story' } )" />
+        <div class="header">
+            <h2 class="title">{{ storyInfo.title }}</h2>
+            <div class="img-preview" :style="imgSyle"></div>
+        </div>
+        <div class="main">
+            <div class="item">
+                <label class="label" for="date">Date</label>
+                <p class="info">{{ storyInfo.creationDate | formatDate }}</p>
```
Comment (stephanebisson) [inline src/views/Publish.vue:11]: Very Angular 😉

File: src/views/Publish.vue [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+            <div class="item">
+                <label class="tags" for="tags">Tags</label>
+                <p class="info">Not set yet</p>
+            </div>
+            <div class="item">
+                <label class="Language" for="tags">Language</label>
+                <p class="info">English</p>
+            </div>
+        </div>
+        <PrimaryButton class="confirm-publish" :text="$i18n('btn-publish-wikistory')" :onClick="() => this.$router.push( { name: 'StoryViewer' } )" />
```
Comment (stephanebisson) [inline src/views/Publish.vue:22]: I would prefer avoiding expression like that in template.

File: src/views/Publish.vue (hunk 2 above)
Comment (hueitan) [reply src/views/Publish.vue:11]: well, there is good and bad 😄

File: src/views/Story.vue [hunk 4]
```diff
@@ -1,21 +1,32 @@
 <template>
 <div class="view story">
+    <PrimaryButton class="publish-button" :text="$i18n('btn-publish')" :onClick="onPublish" />
```
Comment (medied) [inline src/views/Story.vue:3]: I think we're supposed to show the publish button only in the last frame, from what I can gather from the figma

File: src/views/Publish.vue [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+    import { mapGetters } from 'vuex'
+    import PrimaryButton from '@components/PrimaryButton.vue'
+    import Navigator from '@components/Navigator.vue'
+
+    export default {
+        name: 'Publish',
+        components: { Navigator, PrimaryButton },
+        methods: {
+            onPublish: function() {
+                this.$router.push( { name: 'StoryViewer' } );
```
Comment (medied) [inline src/views/Publish.vue:36]: 👀

File: src/views/Story.vue (hunk 4 above)
Comment (hueitan) [reply src/views/Story.vue]: Thanks. just check the design, yes you are right! changed. https://github.com/wikimedia/wikistories-prototype/pull/16/commits/613be16739e0a37f611a414eaa56f0e75189231b

File: src/i18n/en.js [hunk 6]
```diff
@@ -6,4 +6,7 @@ export default {
   'search-media-info': '$1 selected',
   'wikimedia-commons': 'Commons',
   'wikipedia': 'Wikipedia',
+  'btn-publish': 'Publish',
+  'btn-publish-wikistory': 'Publish Wikistory',
+  'tags-not-set': 'Not set yet'
```
Comment (hueitan) [inline src/i18n/en.js:11]: @stephanebisson can you do the `fr` translation for these?

Comment (stephanebisson) [reply src/i18n/en.js:11]: Done

**Pull request merged**